src/components/Gallery/styled.ts

```diff
@@ -61,8 +61,21 @@ export const PlaybackText = styled.span`
   margin: 0 30px 0 4px;
 `;
 
-export const ImageCountText = styled.span`
-  margin-left: 12px;
+export const ImageCountText = styled.p`
+  display: inline-block;
+  margin: 0 0 0 12px;
+
+  span {
+    position: absolute !important;
+    height: 1px;
+    width: 1px;
+    overflow: hidden;
+    clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
+    clip: rect(1px, 1px, 1px, 1px);
+  }
+`;
+
+export const ScreenReaderText = styled.span`
```

Comment:
ScreenReaderText is also a prop? is this used elsewhere?

Reply:
Removed, forgot to remove it